lib/resolver.coffee

```diff
@@ -41,6 +42,9 @@ yeomanResolver = require('yeoman-environment/lib/resolver')
 exports.getNodeModulesPaths = ->
 	paths = yeomanResolver.getNpmPaths()
 
+	if os.platform() isnt 'win32'
+		paths.unshift('/usr/local/lib/node_modules')
```

Comment:
Ah, `module.paths` seems to be what you're referring too, however it only parent directories with `node_modules` appended but doesn't list `/usr/local/lib/node_modules` or similar.